crates/fee/src/lib.rs

```diff
@@ -526,8 +526,8 @@ impl<T: Config> Pallet<T> {
         )
     }
 
-    #[cfg_attr(feature = "integration-tests", visibility::make(pub))]
-    fn distribute_vault_rewards(vault_id: &DefaultVaultId<T>, currency_id: CurrencyId<T>) -> DispatchResult {
+    // #[cfg_attr(feature = "integration-tests", visibility::make(pub))]
```

Comment:
This `cfg_attr` comment can be removed altogether if the function is public for the new RPCs